pre_processing/extract_landscape.py

```diff
@@ -0,0 +1,143 @@
+import dlib
+import numpy as np
+import os
+import csv
+import logging
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+PREDICTOR_PATH = dir_path + "/shape_predictor_68_face_landmarks.dat"
+DATA_CSV_FILE = dir_path + '/../data/fer2013.csv'
+IMAGE_SIZE = 48 * 48
+
+TRAIN_END_POINT = 28708
+PUBLIC_TEST_START_POINT = 28709
+PUBLIC_TEST_END_POINT = 32297
+PRIVATE_TEST_END_POINT = 32297
+
+RIGHT_BROW = list(range(17, 22))
+LEFT_BROW = list(range(22, 27))
+NOSE = list(range(27, 35))
+RIGHT_EYE = list(range(36, 42))
+LEFT_EYE = list(range(42, 48))
+MOUTH = list(range(48, 61))
+
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p',
+                    level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def show_example():
+    """
+    Prints vectors found for first photo in file and shows picture with vectors painted on photo
+    """
+    photos_matrix = _extract_photos_from_file(True)
+    detector = dlib.get_frontal_face_detector()
+    predictor = dlib.shape_predictor(PREDICTOR_PATH)
+    img = photos_matrix[0, :, :]
+    win = _initilise_gui_window(img)
+    dets = detector(img, 1)
+    vec = _get_vectors_of_image_from_image(dets, predictor, img, win)
+    logger.info(vec)
+    win.add_overlay(dets)
+    dlib.hit_enter_to_continue()
+
+
+def get_facial_vectors(specific_part=None, file_path=PREDICTOR_PATH, only_train_data=False, only_test_data=False):
```

Comment:
specific part - adding in code now so will return vectors only of specified Facial parts
file_path - added use in now so used, ability to switch csv file of photos to target